Service/LoginService.robot

```diff
@@ -4,5 +4,5 @@ Resource          ../Page/NavigatorPage.robot
 
 *** Keywords ***
 Login And Wait Top Logo Image UI
-    Login
+    Login    ${ADMIN_LOGIN}    ${ADMIN_PASSWORD}
```

Comment:
What if you will need to login in with non-admin user? Use [Arguments] instead of hardcoded ${ADMIN_LOGIN} and ${ADMIN_PASSWORD}. Provide this data from a test